site/public/js/admin-gradeable-updates.js

```diff
@@ -241,7 +306,39 @@ function ajaxCheckBuildStatus() {
         }
     });
 }
-
+function setRandomGraders(gradeable_id,p_values,successCallback,errorCallback)
+    {
+    var number_to_grade=$('#number_to_peer_grade').val();
+    var gradeable_id=$('#g_id').val();
+    $.ajax({
+        type: "POST", 
+        url: buildCourseUrl(['gradeable', gradeable_id, 'ajaxRandomizePeers']),
+        data: {csrf_token:p_values['csrf_token'],number_to_grade:number_to_grade},
+        success: function(response){
```

Comment:
I think we'll want to parse the response and display it on screen via the peer matrix tab.